src/main/kotlin/no/nav/syfo/api/endpoints/SenOppfolgingEndpoints.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.syfo.api.endpoints
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.syfo.api.model.toResponseDTO
+import no.nav.syfo.application.SenOppfolgingService
+import no.nav.syfo.infrastructure.NAV_PERSONIDENT_HEADER
+import no.nav.syfo.infrastructure.clients.veiledertilgang.VeilederTilgangskontrollClient
+import no.nav.syfo.infrastructure.clients.veiledertilgang.validateVeilederAccess
+import no.nav.syfo.util.getNAVIdent
+import no.nav.syfo.util.getPersonident
+import java.util.*
+
+const val kandidatUuidParam = "vedtakUUID"
+const val senOppfolgingApiBasePath = "/api/internad/v1/senoppfolging"
+const val ferdigbehandlingPath = "/kandidat/{$kandidatUuidParam}/ferdigbehandling"
+
+private const val API_ACTION = "access sen oppfolging kandidat for person"
+
+fun Route.registerSenOppfolgingEndpoints(
+    veilederTilgangskontrollClient: VeilederTilgangskontrollClient,
+    senOppfolgingService: SenOppfolgingService,
+) {
+    route(senOppfolgingApiBasePath) {
+        put(ferdigbehandlingPath) {
```

Comment:
Basert på hva som skjer i db'en så lurer jeg på om post blir hakket mer presist. Hva tenker du om å ha url'en her man initialiserer endepunktet? Synes det blir enklere å lese om url'en er i klartekst der den brukes. Url'er er ikke noe som endres veldig ofte, så synes url'en kan leve i faktisk prodkode og tester hver for seg.
```suggestion
        post(/kandidater/{$kandidatUuidParam}) {
```

Reply:
Ja, tenkte på det. Det er vel en update i kandidat-tabellen og insert i vurdering-tabellen 🤔

Reply:
Enig!